### PR TITLE
Pushover new actions

### PR DESCRIPTION
**Pull request by andrewjschuang**
Pushover new actions

Push Notification | Send a Pushover notification
Emergency Push Notification | Send an emergency Pushover notification

Very cool app!

### CODE REVIEW

**Comment by dylburger** [comment]
- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

File: components/pushover/pushover.app.mjs [hunk 1]
```diff
@@ -1,11 +1,82 @@
...
+      return await axios(this, {
+        method: "post",
+        url: "https://api.pushover.net/1/messages.json",
+        params: {
+          token: `${this.$auth.api_token}`,
+          user: `${this.$auth.user_key}`,
+          ...params,
+        },
+      });
     },
```
[the remark this answers is not in the document]
Comment (andrewjschuang) [reply components/pushover/pushover.app.mjs]: Yeah, I noticed it later but now I've implemented it

**Comment by andrewjschuang** [comment]
> It looks good to me, thanks for these changes!

Np, thanks too for the review! Oh and I was wondering am I the one who merges and drags the issue to the QA column?

**Comment by dannyroosevelt** [comment]
@andrewjschuang looks like these were already merged / published, right?

**Comment by andrewjschuang** [comment]
@dannyroosevelt yeah actually I accidentally merged it before dragging it to QA column

**Comment by dannyroosevelt** [comment]
Got it, no worries.

**Pull request merged**